cpp/ps0002/ps0002_soin_Cpp.cpp

```diff
@@ -1,9 +1,19 @@
-short population = 100;
-short number = 2;
-short tim_people = 20;
-float one_price = 5.5;
-char first_char = 'S';
-long number_births= 14120000;
-short output = 7;
-double invoice = 5555.50;
-long age_ = 536786732663;
+#include <iostream>
+
+using namespace std;
+int main()
+{
+  unsigned short population = 100; // 0.75/0.75
+  unsigned short number = 2; // 0.75/0.75
+  unsigned short tim_people = 20; // 0.75/0.75
+  float pie = 5.5; // 0.75/0.75
+  char first_char = 'S'; // 0.75/0.75
+  unsigned int number_births= 14120000; // 0.75/0.75
+  long double divission = 254435.6547568769; // 0.25/0.75
+  double invoice = 5555.50; // 0.75/0.75
+  unsigned int age_in_seconds = 3786732663; // 0.75/0.75
+  // total 6.25/6.75

```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables